Content.Server/Dice/DiceSystem.cs

```diff
@@ -19,7 +19,14 @@ public override void Roll(EntityUid uid, DiceComponent? die = null)
         if (!Resolve(uid, ref die))
             return;
 
-        var roll = _random.Next(1, die.Sides + 1);
+        int roll;
+
+        if (TryComp<LoadedDiceComponent>(uid, out var loaded) && loaded.SelectedSide != null) {
+            roll = loaded.SelectedSide.Value;
+        } else {
+            roll = _random.Next(1, die.Sides + 1);
+        };
```

Comment:
we usually do
```suggestion
        if (TryComp<LoadedDiceComponent>(uid, out var loaded) && loaded.SelectedSide != null) 
        {
            roll = loaded.SelectedSide.Value;
        } 
        else
        {
            roll = _random.Next(1, die.Sides + 1);
        }
```
tho i'm not sure
and is there a need in semicolon at the end?

Reply:
I'm probably going to refactor this into an event following a discussion in the development channel on Discord, but you're right I just forgot to follow code style for a minute there.